precompile-script.py

```diff
@@ -145,28 +146,34 @@ def convert_stream(fin, fout):
         converter.finish()
 
 
+if __name__ == "__main__":
```

Comment:
If you can be bothered, it might be nice to split this change of adding the `__name == "__main__"` idiom in a separate patch or pull request.